paint.sh

```diff
@@ -16,4 +16,8 @@ do
 		git commit --date="$d 12:$m:$s" -m "$i on $d" --no-gpg-sign --allow-empty
 	done
 done < dates.txt
-git push origin master
+git push $(git origin show) $(git branch --show-current)
```

Comment:
Oops, lost the comment requesting changes: I don't have a `git origin` command.  I confirmed that @BenBE's snippet sets `$remote` correctly when I track something other than origin.

Reply:
pushed a fix, should have been 'git remote show
'

Reply:
`git remote show` lists all the remotes.  Usually there will be only one, especially if people are following the advice not to fork.  But I think the `$remote` trick above is more robust.

Reply:
ah, missed that. made the switch.